requirements/description.rst

```diff
@@ -1,3 +1,41 @@
 Description
 ===========
 
+Product Perspective
+-------------------
+
+Foodtastechess is a web application that is used to play chess.  It will allow a user to play simultaneous games of chess against other remote human players.
+
+
+Web User Interface
+------------------
+
+The web application will have an interface where users will be able to log in and see their active chess games, if any exist.  On this screen, options such as choosing a preferred color may be toggled.  From there, the user will be able to select a game to enter and view the game board.  In a game where it is currently their move, they have the option of taking their turn by moving a piece by either clicking on their piece and then clicking on a valid place to move to, or by dragging the piece there.
+
+
+Product Functions
+-----------------
+
+- Playing a chess game
+- Invite other users to a game
+- Seeing valid moves for a given piece
+- Checking for valid moves
+- Playing multiple games simultaneously
+
+
+User Description
+----------------
+The user in the chess games will be able to sign in with their Google account.  Their profile will feature their picture, which is imported from their Google account.
+
+
+Assumptions	and	Dependencies
```

Comment:
why extra spacing between words? (does this make a difference in the compiled output?)